custom_model_runner/datarobot_drum/drum/lazy_loading/schema.py

```diff
@@ -0,0 +1,64 @@
+from typing import Literal
+from typing import Optional
+
+from pydantic import BaseModel
+from pydantic import conlist
+from pydantic import constr
+from pydantic import model_validator
+
+from datarobot_drum.drum.lazy_loading.constants import BackendType
+
+
+class LazyLoadingFile(BaseModel):
+    remote_path: constr(min_length=1)
+    local_path: constr(min_length=1)
+    repository_id: constr(min_length=1)
+
+    class Config:
+        extra = "ignore"
+
+
+class LazyLoadingRepository(BaseModel):
+    repository_id: constr(min_length=1)
+    bucket_name: constr(min_length=1)
+    credential_id: constr(min_length=1)
+    endpoint_url: Optional[str] = None
+    verify_certificate: Optional[bool] = None
+
+    class Config:
+        extra = "ignore"
+
+    @model_validator(mode="before")
+    @classmethod
+    def validate_model_attributes(cls, data):
+        endpoint_url = data.get("endpoint_url")
+        if endpoint_url is None:
+            return data
+        if endpoint_url == "":
```

Comment:
nit:
```suggestion
        if len(endpoint_url) == 0:
```

Reply:
I think the existing code is more Pythonic and clear because it directly checks an empty string.